torch/distributed/_tensor/device_mesh.py

```diff
@@ -110,13 +110,15 @@ def __init__(
         mesh: Union[torch.Tensor, "ArrayLike"],
         *,
         _init_process_groups: bool = True,
```

Comment:
Can we also delete `_init_process_groups` option in the meantime?

Reply:
Will do in follow up.